readme.md

```diff
@@ -144,6 +147,12 @@ Accepts keywords which ends up as a path in Analytics.
 
 `.track('init', 'backbone')` becomes `/init/backbone`
 
+#### .trackEvent(category, action, [label, value])
```

Comment:
Note that GA supports an options object too:

```js
ga('send', {
  hitType: 'event',
  eventCategory: 'Videos',
  eventAction: 'play',
  eventLabel: 'Fall Campaign'
});
```

https://developers.google.com/analytics/devguides/collection/analyticsjs/events#examples